documentation/src/main/asciidoc/topics/ref_server_ipfilter_cli.adoc

```diff
@@ -0,0 +1,46 @@
+[id='server_endpoint_ipfilter_cli-{context}']
```

Comment:
the content of this file is the same as `documentation/src/main/asciidoc/topics/ref_server_ipfilter.adoc`
you will end up with sections 4.6.1 & 4.6.2 with the same content. 
is this file updated?
shouldn't be the CLI commands?

Reply:
I've written a CLI-specific paragraph now.